code/modules/mob/living/carbon/human/death.dm

```diff
@@ -47,12 +47,18 @@
 		ticker.mode.update_cult_icons_removed(mind)
 		ticker.mode.cult -= mind
 
-	species = null
+	if(species)
+		qdel(species)
+		species = null
```

Comment:
If it was it wouldn't be holding a reference to my mob as my reference testing reveals!

(We have species singletons but they are used to fill in information and do some global things but each mob still has their own species iirc?)